<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1049</width>
    <height>688</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#33363b; font-size:12px; font-family:&quot;Open Sans Bold&quot;;background-color:#fbf9f6;background-repeat:no-repeat;</string>
  </property>
  <widget class="QFrame" name="Detailsframe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>245</y>
     <width>332</width>
     <height>188</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>332</width>
     <height>188</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>332</width>
     <height>188</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/images/detail);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="styleSheet">
       <string notr="true">background:transparent;color:#508f6f;</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>161</width>
         <height>145</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelUnconfirmed">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>33</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Total of transactions that have yet to be confirmed, and do not yet count toward the current balance</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;0 CAIx&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelNumTransactions">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Total number of transactions in wallet</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:8pt; color: #000000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelActive">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>38</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Number of total network connections</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:8pt; color: #000000;&quot;&gt;- &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWalletStatus">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>37</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>37</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#d31360;&quot;&gt;Out of sync&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="balanceFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>331</width>
     <height>218</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>331</width>
     <height>218</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>331</width>
     <height>218</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/images/balance);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>138</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;color:#afa6cd;</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>181</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labelBalance">
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Your current balance</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;0 CAIx&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelStake">
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Balance that is currently staking</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;0 CAIx&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelImmature">
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Balance that has not yet matured</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;0 CAIx&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTotal">
          <property name="font">
           <font>
            <family>Open Sans Bold</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Total CAIx in wallet</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#037e8c; font-size:14px; font-family:&quot;Open Sans Bold&quot;;</string>
          </property>
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;0 CAIx&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="Transactionframe">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>0</y>
     <width>600</width>
     <height>490</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>490</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>677</width>
     <height>245</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {background-image:url(:/images/recent);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QListView" name="listTransactions">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>580</width>
      <height>450</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;color:#037e8c; font-size:11.5px; font-family:&quot;Open Sans Bold&quot;; padding-top:25px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
